tilgangservice/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntektservice/TilgangServiceTest.kt

```diff
@@ -25,15 +28,19 @@ class TilgangServiceTest {
 
         val service = TilgangService(testRapid, mockRedis.store)
         val feil = Fail(
-            eventName = EventName.TILGANG_REQUESTED,
-            behov = BehovType.TILGANGSKONTROLL,
             feilmelding = "ikkeno",
-            data = null,
-            uuid = UUID.randomUUID().toString(),
-            forespørselId = null
+            event = EventName.TILGANG_REQUESTED,
+            transaksjonId = UUID.randomUUID(),
+            forespoerselId = null,
+            utloesendeMelding = JsonObject(
+                mapOf(
+                    Key.BEHOV.str to BehovType.TILGANGSKONTROLL.toJson()
```

Comment:
Er det neo mer som kunne vært inne i denne utloesende meldingen? eventname eller fnr feks?

Reply:
Det vil ikke ha noe å si for testen. Jeg prøvde bare å lage noe så likt som den originale testen.